src/qt/optionsdialog.cpp

```diff
@@ -179,6 +187,15 @@ void OptionsDialog::setModel(OptionsModel *_model)
             ui->overriddenByCommandLineLabel->setText(strLabel);
         }
 
+
+#ifdef ENABLE_WALLET
+        if (strLabel.contains("-enableprivatesend")) {
+            bool fEnabled = CPrivateSendClientOptions::IsEnabled();
+            ui->privateSendEnabled->setChecked(fEnabled);
+            ui->privateSendEnabled->setEnabled(false);
```

Comment:
I don't understand why we are setting it to false...

Reply:
Because cmd-line options override UI settings?

Reply:
But then why is it set to false if privatesend is enabled? I think that might be what I don't understand

Reply:
set to false == disable the checkbox

Reply:
and its not if PrivateSend enabled. Its only if "-enableprivatesend" is set on commandline to 0 or 1, then the checkbox gets disabled while it still represents the actual state.

Reply:
Hmm, okay, so probably makes sense to add a comment?
`
// If -enableprivatesend was passed in on the command line, set the "Enable PrivateSend" button to the value given via commandline and disable the checkbox, ie make it unclickable
`

Also, does this work if I use `-noenableprivatesend`?

Reply:
> Also, does this work if I use `-noenableprivatesend`?

Well, thats no valid commandline parameter unless just putting a `no` in front of a parameter is some hidden feature.. so not sure what should work then :D

Reply:
It's some hidden feature :D I use it often

Reply:
it is 😆 ok wait! About the comment, thats pretty clear actually, isnt it?

Reply:
I mean... I didn't find it super clear what was going on, but I am slow sometimes so :shrug: I'm leaning twords adding the comment

Reply:
Ok checking with the global args doesn't seem to work because it gets set in optionsmodel thats what i wanted to do but now just see 680e85bc3e6d121568cfb58916121f7337cae6ce

Reply:
Looks good to me :D